frontend/src/components/GameGallery.js

```diff
@@ -1,10 +1,27 @@
 import GameThumbnail from "./GameThumbnail";
+import '../styles/GameGallery.css'
+import {NavLink} from 'react-router-dom'
+
+const games = ["1", "2", "3", "4", "5", "6", "7", "8"]
```

Comment:
i removed the NavLinks i was using so we can support modal popups instead. I did leave the main route to the HomePage though.

I updated the games list to be a `let games` instead of `const`. Thank you!